### PR TITLE
Updates the project configuration and documentation (#16)

### PR DESCRIPTION
**Pull request by thadouk**
Fixes some issues in the project documentations and configuration. The local setup session is reduced to 10 as 50 was too long and was delaying the onboarding of the parachain.

# Thank you for your Pull Request

AFTER YOU'VE READ THIS NOTICE, FEEL FREE TO REMOVE IT.
Thank you!

Before you submit a PR, please check that:

- [ ] You describe the purpose of the PR, e.g.:
  - What does it do?
  - Highlight what important points reviewers should know about;
  - Indicates if there is something left for follow-up PRs.
- [ ] Documentation updated
- [ ] Business logic tested successfully
- [ ] Verify First, Write Last: In Substrate development, it is important that you always ensure preconditions are met and return errors at the beginning. After these checks have completed, then you may begin the function's computation.

✄ -----------------------------------------------------------------------------

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,15 +42,53 @@ If you already have Rust installed, make sure you're using the latest version by
...
-Once done, finish installing the support software:
+Once done, finish installing the support software and configure your default toolchain:
 
 ```bash
-# Additional rust targets
-rustup target add wasm32-unknown-unknown --toolchain nightly-2020-10-18
+# Install nightly toolchain and use it
+rustup toolchain install nightly-2022-10-18
+rustup default nightly-2022-10-18
+rustup target add --toolchain nightly-2022-10-18 wasm32-unknown-unknown
```
Comment (jrhender) [inline README.md:51]: I think @JGiter @knzeng-e and I found that we needed `nightly-2022-09-22` (rust v1.66) to get compilation to work. However, I think we can leave as is and adjust in another PR as necessary.

**Review by jrhender — APPROVED** [review]
Looks good to me, thanks @thadouk

**Review by JGiter — APPROVED** [review]

**Comment by thadouk** [comment]
@jrhender should this go to main or dev?
since its an update from upstream

**Comment by jrhender** [comment]
> @jrhender should this go to main or dev?
since its an update from upstream

@thadouk thanks for asking. I think merging into `dev` as you have it here makes sense 👍 

I was initially thinking that we would keep `energywebfoundation:main` at the same commit as `Aventus-Network-Service:main` and then have `energywebfoundation:dev` be the branch that is ahead of `main`. But I now see that the `energywebfoundation:main` branch is redundant and we only need one branch on `energywebfoundation`.
